### PR TITLE
Automated BZ1360239

### PR DESCRIPTION
**Pull request by abalakh**
https://bugzilla.redhat.com/show_bug.cgi?id=1360239
```python
py.test tests/foreman/{api,cli,ui}/test_activationkey.py -k 'test_positive_add_custom_product or test_positive_add_redhat_and_custom_products or test_positive_fetch_product_content'
============================= test session starts ==============================
platform darwin -- Python 2.7.13, pytest-3.0.6, py-1.4.32, pluggy-0.4.0
rootdir: /Users/andrii/workspace/robottelo, inifile:
collected 108 items
2017-03-09 17:28:06 - conftest - DEBUG - Found WONTFIX in decorated tests ['1269196', '1245334', '1217635', '1226425', '1156555', '1204686', '1267224', '1103157', '1230902', '1214312', '1079482']

2017-03-09 17:28:06 - conftest - DEBUG - Collected 108 test cases

2017-03-09 17:28:06 - conftest - DEBUG - Deselected test tests.foreman.api.test_activationkey.test_negative_create_with_no_host_limit_set_max due to WONTFIX


tests/foreman/api/test_activationkey.py .
tests/foreman/cli/test_activationkey.py ..
tests/foreman/ui/test_activationkey.py ..

============================= 103 tests deselected =============================
================= 5 passed, 103 deselected in 2926.19 seconds ==================
```

### CODE REVIEW

File: tests/foreman/cli/test_activationkey.py [hunk 1]
```diff
@@ -668,7 +668,6 @@ def test_positive_add_redhat_product(self):
         self.assertEqual(content[0]['name'], REPOSET['rhst7'])
 
     @run_only_on('sat')
-    @skip_if_bug_open('bugzilla', 1360239)
```
Comment (abalakh) [inline tests/foreman/cli/test_activationkey.py]: 6.3 seems to be not affected anymore despite the fact bz is in POST state - api/cli/ui tests are passing so i can't see a reason to skip these test.

File: tests/foreman/api/test_activationkey.py [hunk 2]
```diff
@@ -417,6 +427,60 @@ def test_positive_remove_user(self):
...
+            product=PRDS['rhel'],
+            repo=REPOS['rhst7']['name'],
+            reposet=REPOSET['rhst7'],
+            releasever=None,
+        )
+        rh_repo = entities.Repository(id=rh_repo_id).read()
+        rh_repo.sync()
+        custom_repo = entities.Repository(
+            product=entities.Product(organization=org).create(),
+        ).create()
```
Comment (ldjebran) [inline tests/foreman/api/test_activationkey.py]: perhaps we should add a yum repo url, what do you think?

File: tests/foreman/api/test_activationkey.py [hunk 3]
```diff
@@ -417,6 +427,60 @@ def test_positive_remove_user(self):
...
+            org_id=org.id,
+            product=PRDS['rhel'],
+            repo=REPOS['rhst7']['name'],
+            reposet=REPOSET['rhst7'],
+            releasever=None,
+        )
+        rh_repo = entities.Repository(id=rh_repo_id).read()
+        rh_repo.sync()
+        custom_repo = entities.Repository(
+            product=entities.Product(organization=org).create(),
```
Comment (ldjebran) [inline tests/foreman/api/test_activationkey.py]: Perhaps we should explicitly show the custom repo url, what do you think?

Comment (abalakh) [inline tests/foreman/api/test_activationkey.py]: @ldjebran red hat products are basically uploaded altogether with subscriptions, if the product was created - then it's definitely a custom one, no need to make emphasis with url.

Comment (rochacbruno) [inline tests/foreman/api/test_activationkey.py]: @ldjebran so this is yours to merge!

**Pull request merged**